Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "ClimaAtmos"
 uuid = "b2c96348-7fb7-4fe0-8da9-78d88439e717"
 authors = ["Climate Modeling Alliance"]
-version = "0.28.6"
+version = "0.28.7"
```

Comment:
Should we make it a breaking release?

Reply:
I think a patch release is good enough? I think this way will be faster to propagate in the Coupler

Reply:
Here is the PR I have in the Coupler https://github.com/CliMA/ClimaCoupler.jl/pull/1222

Reply:
I'm fine either way - whatever @juliasloan25 prefers. And I think we should expect `[[deps.ClimaAtmos]]` in Manifest-v1.11.toml to change as well.

Reply:
Sounds good. Lmk what is the least disruptive and relatively fast way to do it :)

Reply:
To follow best practices it should be a minor release since the interface is changing. One reason it would be nice to have it as a minor version is: in the coupler we'll support both this structure and the previous with an if statement (like `if pkgversion(ClimaAtmos) >= 0.29 integrator.p.precomputed.... else integrator.p.precipitation....`). When we eventually stop supporting both, we can set the compat in the coupler to `0.29` instead of specifying the patch `0.28.7`.

This is a pretty minor example (no pun intended), but the "best" practice is to make a minor release